components/script/script_thread.rs

```diff
@@ -517,6 +520,8 @@ unsafe_no_jsmanaged_fields!(TaskQueue<MainThreadScriptMsg>);
 unsafe_no_jsmanaged_fields!(dyn BackgroundHangMonitorRegister);
 unsafe_no_jsmanaged_fields!(dyn BackgroundHangMonitor);
 
+unsafe_no_jsmanaged_fields!(RefCell<Vec<TransitionOrAnimationEvent>>);
```

Comment:
We should derive JSTraceable on TransitionOrAnimationEvent instead.

Reply:
Okay. I've done this an it's led to a small chain of cleanups due to implementing `JSTraceable` for `RefCell`.